### ISSUE TITLE
Edit theme on the phone

### ISSUE THREAD

**Issue by norman784** [Pixate/pixate-freestyle-ios#189]
Its possible to edit the themes on the phone? I've a project that requires that the user can customise some UI elements and will be cool if its possible to do with Freestyle.

**Comment by elvrsn** [comment]
Yes, its possible. Just change the style class of UI elements when the user customise.
